### ISSUE TITLE
check_data_quality: reinstate check on item classification schemes

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/notebooks-ocds#24]
This isn't covered in Pelican

**Comment by jpmckinney** [comment]
@yolile Do you have any memory of this check?

**Comment by yolile** [comment]
Not really, but I suppose it is around checking whether a scheme exists and if it is valid. I'm not sure if this also included checking if the value item/classification/id is indeed part of the declared schema.

**Comment by jpmckinney** [comment]
Aha, so that would fit under the "Ground truth" heading I added to group some existing checks.

**Comment by jpmckinney** [comment]
I think [this notebook](https://colab.research.google.com/drive/1HrTy5QgXKEIDOIIdZt0RhE8vRua9K-EL) contains the cells to be reinstated (basically, it unions the item classifications across tender, award and contract).
